test/test_dgmulti_1d.jl

```diff
@@ -31,14 +31,7 @@ isdir(outdir) && rm(outdir, recursive = true)
                             9.43689570931383e-16,
                             2.220446049250313e-15
                         ])
-    # Ensure that we do not have excessive memory allocations
-    # (e.g., from type instabilities)
-    let
-        t = sol.t[end]
-        u_ode = sol.u[end]
-        du_ode = similar(u_ode)
-        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
-    end
+    @test_allocations(Trixi.rhs!, semi, sol, 1000)
```

Comment:
Can we keep the comment, please?

Reply:
Sure, done in [33e6442](https://github.com/trixi-framework/TrixiShallowWater.jl/pull/113/commits/33e644259ef51bf14a0ea1c7010c5c8f11426eee).